ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/Coverage/Clusters/ClusterDetailsTable.tsx

```diff
@@ -92,10 +97,16 @@ function ClusterDetailsTable({
         }
     }, [searchFilter]);
 
-    function getStatusByClusterId(clusters: ClusterCheckStatus[]): ComplianceCheckStatus | null {
-        const matchingCluster = clusters.find(
-            (clusterCheckStatus) => clusterCheckStatus.cluster.clusterId === clusterId
+    function getMatchingCluster(clusters: ClusterCheckStatus[]): ClusterCheckStatus | null {
+        return (
+            clusters.find(
+                (clusterCheckStatus) => clusterCheckStatus.cluster.clusterId === clusterId
+            ) || null
```

Comment:
What is pro and con of `||` versus `??` nullish coalescing operator?

Reply:
I think `??` would technically be more correct here since I really only care about `null`/`undefined` but since I'm using `.find` I don't think there's any possibility of the two options resulting in a different outcome. I'll still make the change in my follow up